### PR TITLE
Please

### PR DESCRIPTION
**Pull request by WWWWW-Git**
## About The Pull Request

Blow this gun so far up your fucking ass you'll be speaking darts

## Why It's Good For The Game

Fuck

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/game/objects/items/rogueweapons/ranged/blowgun.dm [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+		if(mastermob.get_num_arms(FALSE) < 2)
+			return FALSE
+		if(mastermob.get_inactive_held_item())
+			return FALSE
+	return TRUE
+
+/datum/intent/arc/blowgun/prewarning()
+	if(mastermob)
+		mastermob.visible_message("<span class='warning'>[mastermob] takes a deep breath!</span>")
+		playsound(mastermob, pick('sound/combat/Ranged/bow-draw-01.ogg'), 100, FALSE)
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueweapons/ranged/blowgun.dm:82]: I love it when holding my breath sounds exactly like a bowstring being pul — What did ye say, mister Feldsher? Asthma? What is that?

File: code/game/objects/items/rogueweapons/ranged/blowgun.dm [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+			return newtime
+		else
+			return 1
+	return chargetime
+
+/datum/intent/shoot/blowgun/arc/arc_check()
+	return TRUE
+
+/obj/item/gun/ballistic/revolver/grenadelauncher/blowgun/shoot_with_empty_chamber()
+	if(cocked)
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueweapons/ranged/blowgun.dm:106]: Goodness these are all crossbow procs still please I just please beg you to make it shoot like a bow or at least test your code **_please_**

File: code/_compile_options.dm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// #define TESTING				//By using the testing("message") proc you can create debug-feedback for people with this
+//#define TESTING				//By using the testing("message") proc you can create debug-feedback for people with this
```
Comment (NPC1314) [inline code/_compile_options.dm:1]: this tho, kinda sinful

**Review by NPC1314 — APPROVED** [review]
It is borderline bloat, theres no real need for this, but hopefully it wont just be a gimmick used once then discarded.

Its kinda cute.

**Pull request merged**